scholia/app/templates/index.html

```diff
@@ -84,6 +84,14 @@ <h4 class="my-0 font-weight-normal">Profiles</h4>
 		    </dd>
 
 		    <dt>
+			<a href="topic/Q84263196">COVID-19</a>
+		    </dt>
+		    <dd>
+			View information about a topic, e.g.
```

Comment:
This should say "View information about the authors or journals publishing on COVID-19", as that is what is linked

Reply:
I updated for COVID and Zika